app/referee/tasks.py

```diff
@@ -166,25 +245,51 @@ def run_submission(submission_id, element_ids, conditional, is_public=True):
     return (submission_run_id, delayeds)
 
 
+def _save_prediction(challenge, evaluation, output_type, raw_value):
+    output_type_model = output_type.content_type.model_class()
+    value_object = output_type_model.from_string(raw_value.strip())
+    value_object.save()
+    models.Prediction.objects.create(
+        challenge=challenge,
+        value_type=output_type,
+        evaluation=evaluation,
+        value_object=value_object,
+    )
+
+
 @dask.delayed(pure=False)
 def run_element(submission_id, element_id, submission_run_id, is_public):
     submission = models.Submission.objects.get(pk=submission_id)
     challenge = submission.challenge
     submission_run = submission.submissionrun_set.get(pk=submission_run_id)
     element = challenge.inputelement_set.get(pk=element_id, is_public=is_public)
 
-    output_type = models.ValueType.objects.get(challenge=challenge, is_input_flag=False)
+    if challenge.valuetype_set.filter(is_input_flag=True).count() == 1:
```

Comment:
Now that I've seen it a few places, I'm wondering if we should store the input and output arg handling in the database instead of checking the length, thoughts?

I like checking since that means we don't need to enforce any consistency/don't need someone making the challenge to make sure they select the right option to prevent errors. Actually what if on challenge creation, we check to see the length of the inputs/outputs and then store `SIMPLE` or `MULTI` in the db?